docs/apis/commands.md

```diff
@@ -104,6 +120,26 @@ apis:
         split_by: \s+
 ```
 
+As stated before, in Windows, you can also override the default shell, `cmd`, with, for example, powershell.
+
+```yaml
+# Used to get list of windows services
+name: windowsServiceList
+apis:
+  - name: windowsServiceList
+    commands:
+      - run: service
+        shell: powershell
+        event_type: WindowsCommand
+        split: horizontal
+        set_header: [status,name,displayname]
+        row_start: 1
+        regex_match: true
+        split_by: (\w+)\s+(\w+)\s+(\w+)
```

Comment:
would highly recommend updating this to a more "PowerShell Native" option; `service` is an Alias for `Get-Service` and there's also a risk that the default output could get messy on longer service names that are automatically truncated in the console.

ex: 
![image](https://user-images.githubusercontent.com/30831476/80512263-d99ccc00-8942-11ea-8c9e-b73e65c872a7.png)


recommended update: 
`- run: Get-Service | Format-Table -Property Status, Name, DisplayName -Autosize`

![image](https://user-images.githubusercontent.com/30831476/80513113-3056d580-8944-11ea-89ec-6debf1cb88bd.png)


alternatively, it might be beneficial to show how [conversion to JSON](https://github.com/newrelic/nri-flex/blob/master/examples/flexConfigs/windows-service-status-example.yml) will remove the need for the majority of this example

Reply:
I changed the command to the native option. But looking at the example of converting to Json, to be honest it seems a lot easier to process it this way than to use powershell to output in json. Using Flex to process the data also emphasizes the "power" of Flex, which is the simplicity. That powershell example is a "beast" and not many people will ever do that kind of thing.

Reply:
I'm ok with that; "beast mode" can be left in the docs